docs/source/nas/space_hub.rst

```diff
@@ -0,0 +1,155 @@
+| MobileNetV3Space  | cream-114              | ImageNet | 72.514  | Test image size = 160         |
++-------------------+------------------------+----------+---------+-------------------------------+
+| MobileNetV3Space  | cream-287              | ImageNet | 77.52   | Default                       |
++-------------------+------------------------+----------+---------+-------------------------------+
+| MobileNetV3Space  | cream-481              | ImageNet | 79.078  | Default                       |
++-------------------+------------------------+----------+---------+-------------------------------+
+| MobileNetV3Space  | cream-604              | ImageNet | 79.92   | Default                       |
++-------------------+------------------------+----------+---------+-------------------------------+
+| DARTS             | darts-v2               | CIFAR-10 | 97.37   | Default                       |
++-------------------+------------------------+----------+---------+-------------------------------+
+| ShuffleNetSpace   | spos                   | ImageNet | 74.14   | BGR tensor; no normalization  |
++-------------------+------------------------+----------+---------+-------------------------------+
+
+.. note::
+
+   1. The metrics listed above are obtained by evaluating the checkpoints provided the original author and converted to NNI NAS format with `these scripts <https://github.com/ultmaster/spacehub-conversion>`__. Do note that some metrics can be higher / lower than the original report, because there could be subtle differences between data preprocessing, operation implementation (e.g., 3rd-party hswish vs ``nn.Hardswish``), or even library versions we are using. But most of these errors are acceptable (~0.1%).
+   2. The default metric for ImageNet and CIFAR-10 is top-1 accuracy.
+   3. The default evaluation follows the protocols in `timm <https://github.com/rwightman/pytorch-image-models>`__.
+
+.. todos: measure latencies and flops, reproduce training.
+
+Searching within model spaces
+-----------------------------
+
+To search for a new architecture on a particular dataset, you can use the following code.
+
+.. code-block:: python
+
+   # Create the model space
+   from nni.retiarii.hub.pytorch import MobileNetV3Space
+   model_space = MobileNetV3Space()
+
+   # Pick a search strategy
+   from nni.retiarii.strategy import Evolution
+   strategy = Evolution()  # It can be any strategy, including one-shot strategies.
+
+   # Define an evaluator
+   from nni.retiarii.evaluator.pytorch import Classification
+   evaluator = Classification(train_dataloaders=DataLoader(train_dataset, batch_size=batch_size),
+                              val_dataloaders=DataLoader(test_dataset, batch_size=batch_size))
+
+   # Launch the experiment, start the search process
+   experiment = RetiariiExperiment(model_space, evaluator, [], strategy)
+   experiment.run(experiment_config)
```

Comment:
what is the next after finishing the search, how can I find the best model on my task?

Reply:
I don't think I should write export and retrain every time. It's already in every tutorial and will be over-tedious.

Reply:
it would be very user friendly if you mention how to retrieve the best model. if you don't want to write it twice, at least add a link to the example code piece.